src/invoke.rs

```diff
@@ -250,11 +268,27 @@ impl Cmd {
             }
         }
 
-        for (i, event) in events.0.iter().enumerate() {
-            eprintln!("Event #{}:", i);
-            match event {
-                HostEvent::Contract(e) => eprint!("{}", serde_json::to_string(&e).unwrap()),
-                HostEvent::Debug(e) => eprint!("{}", e),
+        if self.events {
+            let mut out: Box<dyn io::Write> = Box::new(io::stdout());
+            if let Some(f) = &self.events_file {
+                let file = OpenOptions::new()
+                    .create_new(true)
+                    .write(true)
+                    .append(true)
+                    .open(f)
+                    .map_err(|e| Error::CannotOpenEventsFile {
+                        filepath: f.clone(),
+                        error: e,
+                    })?;
+                out = Box::new(file);
+            }
+
+            for event in events.0 {
+                let event_str = match event {
+                    HostEvent::Contract(e) => serde_json::to_string(&e).unwrap(),
+                    HostEvent::Debug(e) => format!("{}", e),
+                };
+                writeln!(out, "{}", event_str).map_err(|e| Error::CannotWriteEvent { error: e })?;
```

Comment:
This looks like it will produce a file that contains invalid JSON, because debug events are not being outputted as JSON, so I don't think this file will be very usable.

It's quite unclear what the use of writing the events to file are to me, and I think we could leave it out of the CLI.

Reply:
I wrote the intention in the PR's description.

Reply:
I should transform the debug events to JSON though